html/browser/preferences_modern.css

```
@@ -1613,3 +1646,123 @@ body {
+}
+
+.csetup-root.csetup-decoration-minimal .csetup-quirk-row:hover,
+.csetup-root.csetup-decoration-minimal .csetup-quirk-row.is-selected,
+.csetup-root.csetup-decoration-minimal .csetup-quirk-row.is-locked {
+	transform: none !important;
+	box-shadow: none !important;
+}
+
+/* Quirk indicator bar — flat solid color instead of gradient */
+.csetup-root.csetup-decoration-minimal .csetup-quirk-row::before {
+	background: currentColor !important;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)